changelogs/0.1.22.md

```diff
@@ -0,0 +1,15 @@
+Date: 20-02-2022
+
+Tag: 0.1.22
```

Comment:
There are more changes:
- https://github.com/tarantool/vshard/commit/926945756dfc503ca71cb5b36dcf44d918a8c9b4 and https://github.com/tarantool/vshard/commit/b6b95d0ec4da68da16abfcf0d9fe24e6cf58aa04;
- https://github.com/tarantool/vshard/commit/b47b6c29a7bf5e63af89138730402bab34965a67;
- https://github.com/tarantool/vshard/commit/184f640f6f4ee8cecbd6ec17ec7bcd5c6be3db28;
- https://github.com/tarantool/vshard/commit/b01586599f063df1145b48104241a3a9f3e699de;
- https://github.com/tarantool/vshard/commit/d9b2455a741f4096b2ac06fe50aa9afb2ded0c6a;
- https://github.com/tarantool/vshard/commit/0cb510237c1b787472846240db43369e4861c22d and https://github.com/tarantool/vshard/commit/f5f386a5a35e6e5efd8f4f2ed1b3d208fdae9095;

You can see the full list here: https://github.com/tarantool/vshard/compare/0.1.21...master.

Reply:
- #140 didn't have any real cases - so I don't think it's reasonable to put 9269457 and b6b95d0 here.
- b47b6c2  has no associated issue and the behavior is named 'undocumented'. Added anyways.
- 184f640 #173 is mentioned in the rebalancing inconsistency fix 
- b015865 ditto
- d9b2455 mentioned as #107 
- 0cb5102 and f5f386a ditto